frontend/src/shared/components/markerIndex/MarkerIndex.tsx

```
@@ -50,7 +50,7 @@ function MarkerIndex({
         {isRecommended ? (
           <span css={[typography.h3, marker.circle_font()]}>{index}</span>
         ) : (
-          <img src={IconUser} alt="icon-marker-user" draggable={false} />
+          <img src={IconUser} draggable={false} />
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)